client/signup/config/flows-pure.js

```diff
@@ -133,11 +126,13 @@ export function generateFlows( {
 				'plans',
 			],
 			destination: getChecklistDestination,
-			description: 'The improved onboarding flow.',
-			lastModified: '2019-06-05',
+			description: 'The default onboarding flow',
+			lastModified: '2019-06-18',
 		},
 
-		'onboarding-dev': {
+		// We are keeping this flow for now while we
+		// ensure that there are no direct links to /start/onboarding
```

Comment:
Just a Q: Would this not default back to `main` if `onboarding` wasn't found? Is there a negative doing it that way?

Reply:
Hmmm, good question.

 I think you're right. Just tested on https://wordpress.com/websites/create/restaurant/ 

These pages are the only place I know where we hardcode direct links.

If you think it's okay to remove, then we can do it and follow it up with a patch to correct the links shortly after anyway.

👍

Reply:
As it was brought up recently, we should start defaulting back to `onboarding` instead of `main` @spen @ramonjd